model/logs/events.yaml

```diff
@@ -9,24 +9,7 @@ groups:
         type: string
         requirement_level: required
         brief: >
-          The name identifies the event.
-        examples: ['click', 'exception']
-      - id: domain
-        brief: >
-          The domain identifies the business context for the events.
-        type:
-          allow_custom_values: true
-          members:
-            - id: browser
-              value: 'browser'
-              brief: 'Events from browser apps'
-            - id: device
-              value: 'device'
-              brief: 'Events from mobile apps'
-            - id: k8s
-              value: 'k8s'
-              brief: 'Events from Kubernetes'
-        requirement_level: required
-        note: >
-          Events across different domains may have same `event.name`, yet be
-          unrelated events.
+          The name identifies the event and includes a prefixed namespace. The namespace
+          and name are separated by a period. Events across different namespaces may have
+          the same name, yet be unrelated events.
+        examples: ['browser.pointer.click', 'device.exception']
```

Comment:
Sorry for being late on/having missed the discussion (that started in https://github.com/open-telemetry/opentelemetry-specification/issues/2994).

But, with that change I'm wondering how users will be able to filter / query data by the `name` part only?

Let's say I want to see all `exception` events. Now, it's quite simple with `event.name = exception`.
With that change all of this will change to `device.exception`, `browser.exception`, `mycustom.exception`, etc. So how to get ALL types of exceptions?

Also, what is the implication for the [exceptions semantic conventions on span events](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md#recording-an-exception)?:

> An exception SHOULD be recorded as an Event on the span during which it occurred. The name of the event MUST be "exception".

Would that be a plain `exception`, or any prefix expected?

Reply:
Good catch, thanks @AlexanderWert 

The way https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md#recording-an-exception is defined it appears `exception.` is the namespace, not the event name (e.g. we have `exception.message` as the attribute).

I think that aligns nicely with this PR.

We may need to discuss if `exception.` is the the right root name for the namespace or we want to change it.

Reply:
@tigrannajaryan Not sure I understand correctly.

But in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md#recording-an-exception it's written:

> The name of the event MUST be "exception".

So, I'm not referring to the namespace `exception.*`, but on the fact that currently the `event.name` is being set to the value `exception` (and `event.domain` to the domain in which the exception occurred, I guess?). And I'm wondering what the value for `event.name` would be for exceptions with this change here.

Reply:
The event name uniquely defines the type / class of thing that happened. The namespace portion of the event name is meant to avoid collisions, since we may eventually define multiple events called "click" with different semantic meanings. 

For the span event exception API, we'll need to define how the existing semantics of "a span event with name 'exception'", map to the event API and the event semantic conventions. Essentially, when recordException is called, we'll need to define what to assign to the event.name field, and which fields are included in the payload. The payload of record exception seems self explanatory. The event.name field should probably include "exception" as the suffix, but we need to choose an appropriate namespace. I don't think we want to have different namespaces for different environment (i.e. we don't want device.exception, browser.exception, etc). Instead, we should define a environment / runtime agnostic way of indicting that an exception occurred. Maybe the name is something like `runtime.exception` or something similar?

Reply:
One possibility that a separate `event.domain` enabled was to route all events from different domains to different destinations, for eg., different indices in data stores. With the domain merged as namespace into the name, it's still doable but with some complexity. If browser exceptions will not have browser as prefix, I guess we will have to use `Resource` for the purpose.

Reply:
>If browser exceptions will not have browser as prefix, I guess we will have to use Resource for the purpose.

Yes. One kink here is that none of [`browser.`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/browser.md) Resource attributes are "Required" so you can't 100% rely on the presence. I don't know why isn't at least one browser attribute required, maybe it should.

Reply:
I submitted https://github.com/open-telemetry/semantic-conventions/issues/511 to make at least one browser attribute required.

Reply:
Pretty much what most have said here, but in practice I would not "expect" anyone to 

> Let's say I want to see all exception events.

Unless they are the "same" common thing (which is probably unlikely for this example), but if they are the same then it should be the same `event.name`. 

Simplistically, a defined `event.name` (fully qualified definition of an `event`) should be unique and it should (generally) not make any sense to group directly with other `event.name`s. The exception to this would be if specific events are explicitly related or someone is trying to group them based on the same session.id etc. At which point searching / linking based on (only) the name would probably not be possible anyway.

Reply:
Is there any action items from this discussion that need to be accounted for before merging? If so, can those be turned into issues please?